app/View/ListUsersForm.cs

```diff
@@ -1,20 +1,55 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Data;
-using System.Drawing;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using Controller;
+using System;
 using System.Windows.Forms;
 
+
 namespace View
 {
     public partial class ListUsersForm : Form
     {
-        public ListUsersForm()
+        LayoutForm layout_form;
+        public ListUsersForm(LayoutForm layout_form)
         {
             InitializeComponent();
+            this.layout_form = layout_form;
+            UserController.fillDataGridView(dataGridView1);
+            AddActionsToDGV(this.layout_form.auth.permission_id);
+        }
+        private void dataGridView1_CellClick(object sender, DataGridViewCellEventArgs e)
+        {
+            int id = Int32.Parse(dataGridView1.Rows[e.RowIndex].Cells[2].Value.ToString());
+            if (this.layout_form.auth.user_id == id)
+            {
+                ShowWarningMessage();
+            }
+            else
+            {
+                ShowConfirmationMessage(e);
+            }
```

Comment:
(this.layout_form.auth.user_id == id) ? ShowWarningMessage() : ShowConfirmationMessage(e);

Reply:
No se puede con llamadas a métodos en C#